torch/_utils.py

```diff
@@ -494,7 +495,7 @@ def _import_dotted_name(name):
 
 
 # Taken from python 3.5 docs
-def _accumulate(iterable, fn=lambda x, y: x + y):
+def _accumulate(iterable, fn=operator.add):
```

Comment:
This is going to conflict with your other PR in a sec ;)

Reply:
Yeah, I know. ;-;